appengine/standard_python37/pubsub/main_test.py

```diff
@@ -37,20 +80,35 @@ def test_post_index(client):
     assert r.status_code == 200
 
 
+def _verify_oauth2_token_patch():
+    real_verify = jwt.decode
+
+    def mock_verify():
+        real_verify(factory, certs=PUBLIC_CERT_BYTES, verify=True)
+
+    return mock.patch('id_token.verify_oauth2_token', new=mock_verify)
+
+
 def test_push_endpoint(client):
     url = '/_ah/push-handlers/receive_messages?token=' + \
         os.environ['PUBSUB_VERIFICATION_TOKEN']
 
-    r = client.post(
-        url,
-        data=json.dumps({
-            "message": {
-                "data": base64.b64encode(
-                    u'Test message'.encode('utf-8')
-                ).decode('utf-8')
-            }
-        })
-    )
+    r = None
+    with _verify_oauth2_token_patch():
+        r = client.post(
+            url,
+            data=json.dumps({
+                "message": {
+                    "data": base64.b64encode(
+                        u'Test message'.encode('utf-8')
```

Comment:
(minor)
This could simply be written as `b'Test message'`

Reply:
I think it's good to show it this way so it applies to a more general case.